data/pathfinder/paizo/player_companion/animal_archive/anar_abilities_class.lst

```diff
@@ -19,7 +19,7 @@ Animal Trainer		KEY:Carnivalist ~ Animal Trainer		CATEGORY:Special Ability	TYPE:
+Hunting Pack			KEY:Huntmaster ~ Hunting Pack				CATEGORY:Special Ability	TYPE:Extraordinary.ClassFeatures.CavalierClassFeatures.SpecialQuality	DESC:At 1st level, a huntmaster forms a close bond with an animal companion. This animal companion functions as a druid's animal companion, using the cavalier's level as his effective druid level. He can choose either a bird or a dog when selecting his animal companion. The huntmaster can have more than one animal companion, but he must divide up his effective druid level between his companions to determine the abilities of each companion. Each time a huntmaster's effective druid level increases, he must decide how to allocate the increase among his animal companions (including the option of adding a new 1st-level companion). Once an effective druid level is allocated to a particular companion, it cannot be redistributed while that companion is in the huntmaster's service (he must release a companion or wait until a companion dies to allocate its levels to another companion). The share spells animal companion ability does not give the huntmaster the ability to cast a single spell so that it affects all of his animal companions. A huntmaster's animal companions gain a bonus on Survival checks to follow tracks equal to i/2 the cavalier's class level (minimum i). This ability replaces mount.	DEFINE:HuntmasterCompanion|0	BONUS:VAR|HuntmasterCompanion|classlevel("Cavalier")	DEFINE:AnimalCompanionMasterLVL|0		BONUS:VAR|AnimalCompanionMasterLVL|classlevel("Cavalier")	FOLLOWERS:Animal Companion|1	COMPANIONLIST:Animal Companion|Companion (Dog),Companion (Bird (Eagle)),Companion (Bird (Hawk)),Companion (Bird (Owl))																																					SOURCEPAGE:p.8
```

Comment:
First, good job. On the whole this looks great. 

I suspect this will need to be altered a bit to support multiple companion, following the Beastmaster.